templates/react-native/src/enums/enum.ts.twig

```diff
@@ -0,0 +1,6 @@
+export enum {{ enum.name | caseUcfirst  }} {
+{% for value in enum.enum %}
+{% set key = enum.keys is empty ? value : enum.keys[loop.index0] %}
+    {{ key | replace({'-': ''}) | caseEnumKey }} = '{{ value }}',
```

Comment:
`caseEnumKey` should handle doing `replace({'-': ''})` internally

Reply:
Yeah I think it already does.